src/frontend/web_application/src/modules/notification/selectors/messageNotificationsSelector.js

```diff
@@ -1,6 +1,6 @@
 import { createSelector } from 'reselect';
 
-const messageEmmitters = ['smtp', 'twitterBroker'];
+const messageEmmitters = ['smtp', 'twitterBroker', 'mastodonBroker'];
```

Comment:
@sapiens-sapide, if you already started the notifier, is this the name you set for the messageEmmitter ?